docs/src/faq.md

```diff
@@ -62,7 +62,7 @@ argument. Continuing the previous example:
 #] add RandomNumbers
 using RandomNumbers
 jprob = JumpProblem(dprob, Direct(), maj,
-                    rng = Xorshifts.Xoroshiro128Star(rand(UInt64)))
+    rng = Xorshifts.Xoroshiro128Star(rand(UInt64)))
```

Comment:
@ChrisRackauckas @YingboMa is this really what the style is supposed to do? This reformat to a single tab spacing makes it look like an incorrectly indented independent line instead of a function argument...

Reply:
if yas_style_nesting is set to true it'll indent to the function argument